### PR TITLE
several fixes

### PR DESCRIPTION
**Pull request by aszenz**
Fixed one migration, serialized dates to iso as expected by the admin dashboard frontend

### CODE REVIEW

File: migrations/20210310224007_remove_dns_confirmation_code/migration.sql [hunk 1]
```diff
@@ -7,5 +7,5 @@
 
 */
 ALTER TABLE "sites" DROP COLUMN "dns_tag",
-DROP COLUMN "verified",
-DROP INDEX "dns_tag_unique";
+DROP COLUMN "verified";
+DROP INDEX IF EXISTS "dns_tag_unique";
```
Comment (supermacro) [inline migrations/20210310224007_remove_dns_confirmation_code/migration.sql:11]: Hey, these migrations are autogenerated. Was there an issue when you tried running these?

**Review by supermacro — CHANGES_REQUESTED** [review]

File: migrations/20210310224007_remove_dns_confirmation_code/migration.sql (hunk 1 above)
Comment (aszenz) [reply migrations/20210310224007_remove_dns_confirmation_code/migration.sql:11]: Yes migrations failed, the syntax is wrong dropping an index is a global operation and doesn't fit inside an alter table statement

File: src/routes/serialize.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { camelCase, isObject } from 'utils'
 import { JSONObject, JSONValues } from 'router'
 
-const dateSerializer = (date: Date): number => date.getTime()
+const dateSerializer = (date: Date): string => date.toISOString()
```
[the remark this answers is not in the document]
Comment (aszenz) [reply src/routes/serialize.ts:4]: Alright, I guess then we don't need the elm-iso package anymore too.

File: migrations/20210310224007_remove_dns_confirmation_code/migration.sql (hunk 1 above)
Comment (aszenz) [reply migrations/20210310224007_remove_dns_confirmation_code/migration.sql:11]: after researching a bit more turns out when u drop a column in postgres, the index is automatically dropped, probably the reason i had to use `if exists` then

File: src/routes/serialize.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (aszenz) [reply src/routes/serialize.ts]: done in https://github.com/parlez-vous/admin-dashboard/pull/67/commits/09d94c1c782ec9d3bfd081df501438727c536c51

**Pull request merged**